nautobot/core/templates/inc/media.html

```diff
@@ -3,8 +3,8 @@
 {% load plugins %}
 
     <link rel="stylesheet"
-          href="{% static 'bootstrap-3.4.1-dist/css/bootstrap.min.css' %}"
-          onerror="window.location='{% url 'media_failure' %}?filename=bootstrap-3.4.1-dist/css/bootstrap.min.css'">
+          href="{% static 'bootstrap-3.4.1-dist/css/bootstrap.css' %}"
+          onerror="window.location='{% url 'media_failure' %}?filename=bootstrap-3.4.1-dist/css/bootstrap.css'">
```

Comment:
Is this a temporary change just for development?

Reply:
Yeah, because for `bootstrap.min.css` is not formated and very hard to tell the difference between changes